openslides/core/static/js/core/site.js

```diff
@@ -197,6 +197,29 @@ angular.module('OpenSlidesApp.core.site', [
                 data: {extern: true},
                 onEnter: function($window) {
                     $window.location.href = this.url;
+                },
+                resolve: {
```

Comment:
As it seems, this section is not even triggered (a console log in onEnter never fires), not when opening the projector directly, and neither when opening it through the core app.